_includes/contact_form.html

```diff
@@ -1,9 +1,4 @@
-{% comment %}
-
-include:
-- common_applications: array of strings (translation keys)
-
-{% endcomment %}
+{% comment %} include: - common_applications: array of strings (translation keys) {% endcomment %}
```

Comment:
Line 370, 371 were the main change. I broke out hours of operation and help text into two separate things, so 1) they can change independently, and 2) so I can reuse hours of operation in the phone number area.

As an aside, I thought we were supposed to use prettier to format files... but it looks like a lot of them weren't prettier formatted?

Reply:
> Prettier formatting is only enforced for specific file types like JavaScript (and YAML in the IdP). I've found Prettier's HTML formatting to be pretty lacking (especially with ERB / Liquid syntax like this) so I typically avoid it for these files.

To that point, I think it'd nice to use Prettier with YAML files in this project, and I don't mind those revisions in this pull request, though it does distract a bit from the intended changes. It could be a good follow-up action item to format the YAML consistently and add some lint enforcement.